src/content/docs/glossary/glossary.mdx

```diff
@@ -635,7 +634,10 @@ The length of time in seconds to wait for the aggregation window to fill with da
     id="integration"
     title="integration"
   >
-    At New Relic, an integration refers to a solution that integrates with a specific technology (like a web framework or a type of database). All our integrations can be found as quickstarts in [New Relic Instant Observability](https://newrelic.com/instant-observability/).
+
+At New Relic, "integration" refers to any software technology that New Relic can make a connection to. So, for example, our agents contain various types of integrations (ways to report data from various app frameworks, or operating systems, or types of database services). Other integrations are configurations that allow a third-party service (for example, AWS Lambda or PagerDuty) to send data to us. 
```

Comment:
@zuluecho9, I'm not sure what you mean by "database services." Our on-host integrations integration with various databases (MongoDB, MySQL, etc.), but when I think of data services, I think of databases as a service, which is a cloud offering where someone else provides you with a database so you don't have to maintain it.

Reply:
yeah, maybe just 'types of databases' instead